config.toml.example

```diff
@@ -563,6 +563,14 @@ changelog-seen = 2
 
 # Use LLVM libunwind as the implementation for Rust's unwinder.
 # Accepted values are 'in-tree' (formerly true), 'system' or 'no' (formerly false).
+# This option only applies for Linux and Fuchsia targets.
+# On Linux target, if crt-static is not enabled, 'no' means dynamic link to
+# `libgcc_s.so`, 'in-tree' means static link to the in-tree build of llvm libunwind
+# and 'system' means dynamic link to `libunwind.so`. If crt-static is enabled,
+# the behavior is depend on the libc. On musl target, 'no' and 'in-tree' both 
```

Comment:
Does musl support linking with static libgcc at all?
If it does, then linking to libunwind when `llvm-libunwind = 'no'` is specified doesn't look right, perhaps rustbuild should have different defaults for musl instead.

I don't suggest doing it in this PR, but could you make an issue for this (if musl does support linking with static libgcc)?
Otherwise adding the fact that musl doesn't support linking with static libgcc to the docs should be enough.

Reply:
> Does musl support linking with static libgcc at all?

musl support linking with `libgcc_eh.a`

>  If it does, then linking to libunwind when llvm-libunwind = 'no' is specified doesn't look right, perhaps rustbuild should have different defaults for musl instead.

This is the default behavior for musl target since `libunwind` crate was added. Maybe https://github.com/rust-lang/compiler-team/issues/422 would fix this situation.